modules/apps/object/object-test/src/testFunctional/tests/ObjectNotifications.testcase

```diff
@@ -120,6 +120,39 @@ definition {
 		}
 	}
 
+	@description = "LPS-161923 - Verify that it's possible to search for notifications"
+	@priority = "3"
+	test CanSearchForNotifications {
+		task ("Go to Templates and add Notification Templates") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Notifications",
+				panel = "Control Panel",
+				portlet = "Templates");
+
+			for (var templateName : list "Test,Liferay,Site") {
+				ObjectNotifications.addNewNotificationTemplate(
+					emailBody = "Email Body",
+					entryTo = "test@liferay.com",
+					fromAddress = "Liferay",
+					fromName = "Test Test",
+					name = "Notification ${templateName}",
+					subject = "Subject");
+
+				Button.clickSave();
+			}
+		}
+
+		task ("Search for Template by name") {
```

Comment:
Search for the Notification Template by name

We should keep our wording consistent throughout our tests.